From: Aymeric Agon-Rambosson <aymeric.agon@yandex.com>
Date: Tue, 26 Dec 2023 18:20:01 +0100
Subject: Replace multiline Package-Requires with monoline

Forwarded: not applicable

dh-elpa is not capable yet of parsing multiline Package-Requires correctly. Work
is being done to add this feature, but we make do with this patch in the
meantime.
---
 lisp/magit-section.el | 6 +-----
 lisp/magit.el         | 9 +--------
 2 files changed, 2 insertions(+), 13 deletions(-)

diff --git a/lisp/magit-section.el b/lisp/magit-section.el
index ba2dbf1..c39f613 100644
--- a/lisp/magit-section.el
+++ b/lisp/magit-section.el
@@ -9,11 +9,7 @@
 ;; Keywords: tools
 
 ;; Package-Version: 4.1.0
-;; Package-Requires: (
-;;     (emacs "26.1")
-;;     (compat "30.0.0.0")
-;;     (dash "2.19.1")
-;;     (seq "2.24"))
+;; Package-Requires: ((emacs "26.1") (compat "30.0.0.0") (dash "2.19.1") (seq "2.24"))
 
 ;; SPDX-License-Identifier: GPL-3.0-or-later
 
diff --git a/lisp/magit.el b/lisp/magit.el
index bc3e6f0..ec7d07b 100644
--- a/lisp/magit.el
+++ b/lisp/magit.el
@@ -18,14 +18,7 @@
 ;; Keywords: git tools vc
 
 ;; Package-Version: 4.1.0
-;; Package-Requires: (
-;;     (emacs "26.1")
-;;     (compat "30.0.0.0")
-;;     (dash "2.19.1")
-;;     (magit-section "4.1.0")
-;;     (seq "2.24")
-;;     (transient "0.7.5")
-;;     (with-editor "3.4.2"))
+;; Package-Requires: ((emacs "26.1") (compat "30.0.0.0") (dash "2.19.1") (magit-section "4.1.0") (seq "2.24") (transient "0.7.5") (with-editor "3.4.2"))
 
 ;; SPDX-License-Identifier: GPL-3.0-or-later
 
